offline first: add a hint that the help is also available for relaxed, in-app reading when being offline (#645)
deltachat/deltachat-pages: en/help.md

## What is Delta Chat?

> > In a hurry? You can read this help relaxed at anytime within the app, also without internet.

Reviewer: ```suggestion
> You can also read this help relaxed at anytime within the app, and it works without internet.
```

I would remove the 'hurry', and slightly reword.

Reviewer: Not a blocker
